recipes/security-agent.rb

```diff
@@ -65,14 +73,18 @@
 # Common configuration
 service_provider = Chef::Datadog.service_provider(node)
 
-service_name = 'datadog-agent-security'
+service_name = is_windows ? 'datadog-security-agent' : 'datadog-agent-security'
```

Comment:
This is probably not related to this PR, but why???

Reply:
I have no idea.. I'm just copying what has been done for system probe

Reply:
is the name of the service on linux `datadog-security-agent` or `datadog-agent-security`?
Right now, the name of the service (on windows) is `datadog-security-agent`.  Since we haven't shipped, we can make it `datadog-agent-security` without breaking anything, and then it will match linux if that's desirable.